Code/Framework/AzCore/AzCore/Math/MathUtils.h

```diff
@@ -467,21 +467,13 @@ namespace AZ
     //! Example: alignment: 4
```

Comment:
Recommend documenting here that 0 is an invalid alignment, which will expose the caller to undefined behavior

Reply:
Updated, thanks for the suggestion.